views/treetest/view.ejs

```diff
@@ -8,6 +8,7 @@
 		<!-- custom css -->
 		<link rel="stylesheet" type="text/css" href="/css/treetest.css" />
 		<title>Kort Treetest</title>
+		<script type="text/javascript" src="/js/uuid.min.js"></script>
```

Comment:
Not sure I understand the need for that in the frontend code as I see no use of it in this file? 

Is that really necessary?